src/components/SubCategorySummary/SubCategorySummary.test.js

```diff
@@ -3,12 +3,14 @@ import { shallow } from 'enzyme';
 import CategoryBanner from './SubCategorySummary';
 
 jest.mock('../Card/IndicatorCard/IndicatorCard', () => 'IndicatorCard');
+jest.mock('../Icon/Icon', () => 'Icon');
```

Comment:
Mocking out the `<Icon>` in tests means that the snapshot of a component won't change if the icon component is changed.